AppCenterData/AppCenterDataTests/MSDataTests.m

```diff
@@ -2819,4 +2821,125 @@ - (void)testRemoteOperationDelegateMethodIsCalledWithNoExistenceOfImplementation
                                }];
 }
 
+- (void)testDocumentChangeDelegateMethodIsCalled {
```

Comment:
Please follow `// If`, `// When`, and `// Then` convention.

Reply:
Updated to follow convention